static/js/settings_emoji.js

```diff
@@ -21,6 +21,15 @@ const meta = {
     loaded: false,
 };
 
+export function is_custom_emoji(emoji_name) {
+    const emj_data = emoji.get_server_realm_emoji_data();
+    for (const emj of Object.values(emj_data)) {
+        if (emj.name === emoji_name && !emj.deactivated) {
```

Comment:
Keeping the variable names as `emoji_data` rather than `emj_data` and `emoji` rather than `emj` is better as we also use the same variable name <a href = 'https://github.com/zulip/zulip/blob/master/static/js/settings_emoji.js#L89'>here</a>.

Reply:
I will change it

Reply:
I changed emj to emoji.